apps/console/src/features/identity-providers/components/forms/authenticators/saml-authenticator-form.tsx

```diff
@@ -0,0 +1,558 @@
+/* eslint-disable sort-keys */
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { Field, Form } from "@wso2is/form";
+import { Code, FormInputLabel, FormSection } from "@wso2is/react-components";
+import React, { FunctionComponent, PropsWithChildren, ReactElement, useEffect, useMemo, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Grid, SemanticWIDTHS } from "semantic-ui-react";
+import { AppState, ConfigReducerStateInterface } from "../../../../core";
+import {
+    CommonAuthenticatorFormInitialValuesInterface,
+    FederatedAuthenticatorWithMetaInterface
+} from "../../../models";
+import { composeValidators, fastSearch, getAvailableNameIDFormats, getAvailableProtocolBindingTypes, getDigestAlgorithmOptionsMapped, getSignatureAlgorithmOptionsMapped, hasLength, IDENTITY_PROVIDER_ENTITY_ID_LENGTH, isUrl, LOGOUT_URL_LENGTH, required, SERVICE_PROVIDER_ENTITY_ID_LENGTH, SSO_URL_LENGTH } from "../../utils/saml-idp-utils";
```

Comment:
Possible line length issue here. please check.

Reply:
Addressed this with commit 4ca4768